workspaces/variables.md

```diff
@@ -1,23 +1,24 @@
 ---
-title: Workspace variables
-description: Learn how to work with CODER_* workspace variables
+title: Environment variables
```

Comment:
I think these changes should be in a separate PR to keep the history clean

Reply:
unsure how this got into here in the first place. thanks